packages/block-library/src/comment-template/block.json

```
@@ -1,4 +1,5 @@
 {
+	"$schema": "https://json.schemastore.org/block.json",
```

Schema URL has changed as of https://github.com/WordPress/gutenberg/pull/35900:

```suggestion
	"$schema": "https://schemas.wp.org/trunk/block.json",
```

It should be fixed also in https://github.com/WordPress/gutenberg/pull/36351.